modules/openapi-generator/src/main/resources/go-server/routers.mustache

```diff
@@ -108,7 +108,11 @@ func EncodeJSONResponse(i interface{}, status *int,{{#addResponseHeaders}} heade
 		w.WriteHeader(http.StatusOK)
 	}
 
-	return json.NewEncoder(w).Encode(i)
+    if (i) != nil {
```

Comment:
what about just the following without parenthesis ?
```
if i != null {
```

Reply:
Changed!